os/board/common/partitions.c

```diff
@@ -147,10 +154,16 @@ void configure_partitions(void)
 				}
 			}
 			mtd_setpartitionname(mtd_part, part_name);
+#ifdef CONFIG_BINARY_MANAGER
+			if (!strncmp(types, "bin,", 4)) {
```

Comment:
Where is getting kernel binary information?

Reply:
binary manager reads header in binary_manager_load_bininfo function.

Reply:
Don't you need to get a partition information of kernel like apps?
You only read apps information using "bin,".